db/migrate/20150118173952_add_is_required_to_assignments.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsRequiredToAssignments < ActiveRecord::Migration
+  def change
+    add_column :assignments, :is_required, :boolean, default: 'false'
```

Comment:
I'll repeat my above comment; I think a better column name is something like `only_required_files`.